### PR TITLE
iZotope.ProductPortal version 1.4.3

### PR DESCRIPTION
**Pull request by derkrasseleo**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43594)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/i/iZotope/ProductPortal/1.4.3/iZotope.ProductPortal.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: iZotope.ProductPortal
+PackageVersion: 1.4.3
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://www.izotope.com/in-app/pp/download/pc
+  InstallerSha256: FD350390F44D7D48D7D996F4C35412AAAE007C407BA5ABD4CFBB9CDBAB8EE54B
```
Comment (Trenly) [inline manifests/i/iZotope/ProductPortal/1.4.3/iZotope.ProductPortal.installer.yaml:10]:
```suggestion
  InstallerUrl: https://s3.amazonaws.com/izotopedownloads/product_download/iZotope_Product_Portal_v1_4_3.exe
  InstallerSha256: FD350390F44D7D48D7D996F4C35412AAAE007C407BA5ABD4CFBB9CDBAB8EE54B
  ProductCode: 'Product Portal'
  InstallerSwitches:
    Silent: --mode unattended
    SilentWithProgress: --mode unattended
```

File: manifests/i/iZotope/ProductPortal/1.4.3/iZotope.ProductPortal.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: iZotope.ProductPortal
+PackageVersion: 1.4.3
+PackageLocale: en-US
+Publisher: iZotope
+PackageName: Product Portal
+License: Copyright iZotope, Inc.
```
Comment (Trenly) [inline manifests/i/iZotope/ProductPortal/1.4.3/iZotope.ProductPortal.locale.en-US.yaml:9]:
```suggestion
Publisher: iZotope, Inc.
PackageName: Product Portal
License: Proprietary
Copyright: Copyright iZotope, Inc.
```

File: manifests/i/iZotope/ProductPortal/1.4.3/iZotope.ProductPortal.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: iZotope.ProductPortal
+PackageVersion: 1.4.3
+PackageLocale: en-US
+Publisher: iZotope
+PackageName: Product Portal
+License: Copyright iZotope, Inc.
+ShortDescription: iZotope Product Portal
```
Comment (Trenly) [inline manifests/i/iZotope/ProductPortal/1.4.3/iZotope.ProductPortal.locale.en-US.yaml:10]: Please provide a better description of what the Product Portal does

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/43594#discussion_r794153141

**Comment by leochras** [comment]
> [#43594 (comment)](https://github.com/microsoft/winget-pkgs/pull/43594#discussion_r794153141)

Is it okay now?

**Review by OfficialEsco — APPROVED** [review]
i guess

**Pull request merged**